resources/benchmark-runner.mjs

```diff
@@ -289,9 +287,16 @@ export class BenchmarkRunner {
             await this._client.didRunSuites(this._measuredValues);
         }
     }
+
+    getIterationTotalMetric(i) {
```

Comment:
WebKit's naming convention is not to use "get" prefix in these cases:
https://webkit.org/code-style-guidelines/#names-setter-getter

So this can be just iterationTotalMetric(i)